app/models/spree/order.rb

```diff
@@ -310,6 +310,8 @@ def ship_total
     # Creates new tax charges if there are any applicable rates. If prices already
     # include taxes then price adjustments are created instead.
     def create_tax_charge!
+      return if state.in?(["cart", "address", "delivery"]) && Flipper.enabled?(:split_checkout)
```

Comment:
No need to specify the current spree user here?

Reply:
It's in a model, so this bit of code doesn't necessarily have access to the request object or the current user (via the request's session).

So it's effectivley checking if the whole feature is fully enabled for all users.

Reply:
So if the features is only enabled for one user then we update tax charges even if we are only in the cart state. I guess that's a tiny performance hit but nothing harmful.

Reply:
Ok! Make sense.